ddtrace/contrib/internal/anthropic/patch.py

```diff
@@ -47,7 +47,8 @@ def traced_chat_model_generate(anthropic, pin, func, instance, args, kwargs):
     span = integration.trace(
         pin,
         "%s.%s" % (instance.__class__.__name__, func.__name__),
-        submit_to_llmobs=True,
+        # only report LLM Obs spans if base_url has not been changed
+        submit_to_llmobs=integration.is_default_base_url(str(instance._client._base_url)),
```

Comment:
Let's be defensive here (`getattr()`) and other places where we try accessing instance attributes.